cpc/dataset.py

```diff
@@ -91,6 +91,7 @@ def clear(self):
             del self.seqLabel
 
     def prepare(self):
+        random.seed(767543)
```

Comment:
Shouldn't this use the seed actuall passed in args?

Reply:
I think I'd either add another arg that would be the seed for batching so that it isn't hardcoded, or use hardcoded one, and in either case use separate rng with this seed here to make it independent so that other random stuff can be "more random" without setting main rng's seed to specific value. Wdyt?

Reply:
Let's keep the seed.